numba_dppy/compiler.py

```diff
@@ -545,32 +545,39 @@ def check_for_invalid_access_type(self, access_type):
 
 
 class JitDPPYKernel(DPPYKernelBase):
-    def __init__(self, func, access_types):
+    def __init__(self, func, debug, access_types):
 
         super(JitDPPYKernel, self).__init__()
 
         self.py_func = func
         self.definitions = {}
+        self.debug = debug
```

Comment:
This variable is to forward the debug flag provided in the `@dppy.kernel` decorator. We are not using it right now but we allow users to provide the debug flag in the decorator. I am saving it so that in the future when we decide to implement the desired functionality we can know when users have provided it later in the pipeline.

Reply:
Ok